### PR TITLE
chore(showcase): Add OneDeck to showcase

### PR DESCRIPTION
**Pull request by WillNeill**
Add OneDeck (www.onedeck.co) to site showcase

## Description

Would love to add the new OneDeck site to the Gatsby showcase. We've loved working with Gatsby.

## Related Issues

n/a

### CODE REVIEW

File: docs/sites.yml [hunk 1]
```diff
@@ -5136,3 +5136,14 @@
   built_by: Ariadne Digital
   built_by_url: https://www.ariadnedigital.it
   featured: false
+  - title: OneDeck
```
Comment (lannonbr) [inline docs/sites.yml]:
```suggestion
- title: OneDeck
```

Fixing the yaml formatting

Comment (WillNeill) [reply docs/sites.yml]: Thanks @lannonbr!

Comment (lannonbr) [reply docs/sites.yml]: If you agree with the fix, click that "commit suggestion" button

**Comment by pieh** [comment]
```
docs/sites.yml: SyntaxError: Document is not valid YAML (bad indentation?) (5139:3)
   5137 |   built_by_url: https://www.ariadnedigital.it
   5138 |   featured: false
 > 5139 |   - title: OneDeck
        |   ^^^^^^^^^
   5140 |   main_url: https://www.onedeck.co
   5141 |   url: https://www.onedeck.co
   5142 |   description: >
```

**Pull request merged**